gateway/src/args.rs

```diff
@@ -62,6 +62,9 @@ pub struct ContextArgs {
     /// Address to reach the authentication service at
     #[arg(long, default_value = "http://127.0.0.1:8008")]
     pub auth_uri: Uri,
+    /// Address to reach the resource recorder service at
+    #[arg(long, default_value = "http://resource-recorder:8000")]
+    pub resource_recorder_uri: Uri,
```

Comment:
If we have access to r-r, do we need projects to be running to delete them? I suppose database deletion also goes through the deployer provisioner client, but could that also be done in gateway?

Reply:
On second thought, we do need the deployer to have been started recently, to ensure any resources cached there have been sent to r-r.

Reply:
Oh, gateway won't access r-r. This is in the context struct that we send to our Docker worker. This uri is then sent to a deployer when we start it up. But it is an interesting option though....